README.md

```diff
@@ -63,6 +63,23 @@ Packer and cloudformation won't automatically remove old AMI versions, so this h
 done manually. AMIs can be deleted from the EC2 console by deregestering the AMI and
 deleting the related EBS snapshot.
 
+## Deploying AWS Lambda functions
+
+Lambda functions are created by the CloudFormation stack, but rerunning the stack won't updated
+the function code. Instead, functions can be updating using the `lambda-release` command:
```

Comment:
Could be either. For now, I've done this locally, but we could set up a Jenkins job to deploy new changes after review.